docs/KoreBuild.md

```diff
@@ -0,0 +1,88 @@
+KoreBuild
+=========
+
+KoreBuild is a set of MSBuild targets and tasks used to define a common set of build and test tasks. The entry point for KoreBuild is defined in [KoreBuild.proj][korebuild-proj].
+
+## KoreBuild Lifecycle
+
+The KoreBuild chains together these targets. Custom targets can chain of these
+
+1. Prepare - pre-build actions like making directories
+1. Restore - NuGet restore
+1. Compile - calls /t:Build on \*.sln files
+1. Package - NuGet pack and other packaging steps
+1. Test - invokes VSTest
+1. Verify - post build tests
+1. Build = the default target. Runs Prepare;Restore;Compile;Package;Test;Verify
```

Comment:
This format implies that Build will be run after Verify. If you're just listing them and not trying to imply an order of execution I would suggest a bullet-ed list.

Reply:
updated